docs/building-deployer.md

```diff
@@ -1,13 +1,14 @@
 # Building an application deployer image
 
 This guide covers the steps to build a deployer container image for the Kubernetes
-applications that you distribute on Google Cloud Platform (GCP) Marketplace.
+applications that you distribute on Google Cloud Platform (Google Cloud) Marketplace.
```

Comment:
Should this just be Google Cloud Marketplace?

Reply:
(asking instead of leaving it as a suggested edit because it looks intentional 😅)

Reply:
Yes it should. Find/replace artifact :)